### PR TITLE
Create nl-application.json

### PR DESCRIPTION
**Pull request by johnniezen**
Translation for Dutch language
I have some errors in the file on lines 111~123, 136, 138, 317 and 318. Not sure and able how to fix.

### CODE REVIEW

File: routes/lang/nl-application.json [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+  "app.layoutright": "Rechts",
+  "app.layoutfull": "Volle-breedte",
+  "app.placeholdernewcourse": "Curses titel",
+  "app.placeholdernewpage": "Pagina titel",
+  "app.placeholdernewmenu": "Sectie titel",
+  "app.placeholdernewarticle": "Artikel titel",
+  "app.placeholdernewblock": "Blok titel",
+  "app.placeholdernewcomponent": "Component titel",
+  "app.placeholdereditthistext": "{Verander deze tekst...}",
+  "app.confirmpublish": "Weet u zeker dat  u wilt publiceren?",
```
Comment (tomgreenfield) [inline routes/lang/nl-application.json]: Remove extra space after `Weet u zeker dat`.

File: routes/lang/nl-application.json [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+  "app.exportcoursetext": "Even geduld, dit kan even duren.",
+  "app.exportcoursesuccess": "Cursus successvol geexported!",
+  "app.infodefaulttitle": "Informatie",
+  "app.errordefaulttitle": "Fout",
+  "app.confirmdefaulttitle": "Weet u het zeker?",
+  "app.confirmdefaultyes": "Ja, ik weet het zeker",
+  "app.confirmwait": "Wacht",
+  "app.errorsessionexpired": "Uw sessie is verlopen, klik OK om weer in te loggen",
+  "app.errorassetupdate": "Er is iets fout gegaan terwijl de asset werd geupdate.<br/>Probeer opnieuw.",
+  "app.errordeleteasset": "Kon deze  asset niet verwijderen, %{message}",
```
Comment (tomgreenfield) [inline routes/lang/nl-application.json]: Remove extra space after `Kon deze`.

File: routes/lang/nl-application.json [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+  "app.addsection": "Voeg sectie toe",
+  "app.editthistext": "{ Bewerk deze tekst}",
+  "app.overview": "Overzicht",
+  "app.resetpassword": "Reset wachtwoord",
+  "app.resetpasswordsuccess": "Uw wachtwoord is veranderd",
+  "app.resetpassworderror": "Uw wachtwoord kon niet veranderd worden",
+  "app.submit": "Zenden",
+  "app.newpassword": "Nieuw wachtwoord",
+  "app.forgotpasswordblurb": "Voer het e-mail adres in wat bij uw Learning Pool Adapt account hoort, klik dan op Continue. Wij sturen een email met een link waar u makkelijk een nieww wachtwoord maakt.",
+  "app.forgotpasswordfooter": "Indien u niet langer het e-mail adres wat bij uw  Adapt account hoort gebruikt, neem dan contact op met Support voor hulp bij het herstellen van uw account.",
```
Comment (taylortom) [inline routes/lang/nl-application.json]: Also question the Learning Pool reference here.

Comment (johnniezen) [inline routes/lang/nl-application.json]: I was not sure if it is still valid, so I left it in. Might it be possible to change it with something like 'your organization' ?

**Comment by taylortom** [comment]
Many thanks for the PR John!

Couple of things:
- Which commit did you base the translation on? It doesn't seem to match the latest in the master branch; the references to **Learning Pool** and **Adapt Builder** caught my eye.
- We're very close to releasing a new version of the tool (i.e. days away) which will introduce some new language strings. Would be great if you could add these to your PR.

**Comment by johnniezen** [comment]
Hello Tom, if the strings are already available, please sent them, or point me were to get them and I’ll add them in the current one.

As for where they came from, the file could be a little older, like from September or October, when I wanted to make a first start.
Never really managed to get some minutes to finish.
When I noticed the ES PR recently, I compared that with mine.

If there are any mix ups, or deletions needed, just tell me so.

"Think Quality First, Care to Quality, Focus to Customer, Be the Best"
Kind regards / Met vriendelijke groet, Meilleures salutations /Cordiales saludos, 지원에 감사드리며,
John Niezen
Senior Technical Solution & Training Instructor
LGESC - Europe SVC Department - Service Academy Europe<https://service.academy.lge.com/>
LG Electronics European Shared Service Center B.V.
[cid:image001.png@01D2724E.36FB9A80]<https://play.google.com/store/apps/details?id=com.lgsvcacademy.lgrepairsupport>

**Comment by taylortom** [comment]
Thanks John!

You'll find the latest version of the file here: https://raw.githubusercontent.com/adaptlearning/adapt_authoring/master/routes/lang/en-application.json

You can see the latest amends by looking at the file's blame page: 
https://github.com/adaptlearning/adapt_authoring/blame/master/routes/lang/en-application.json

**Comment by johnniezen** [comment]
Thanks Tom,
I compared the files, and a I could only see the difference in the last 4 lines, which are removed.
Hope it is all good now.

**Comment by taylortom** [comment]
@johnniezen  I've just set up a quick test, and still getting differences (sorry!). Have set up [this gist](https://gist.github.com/taylortom/22dde8fe75dd13a51070f4c356cf628f) which will hopefully be of some help.

**Comment by johnniezen** [comment]
Hi Tom,
You gist was really helpfull. Hope to have captured all differences now.
Thank you and Tom Greenfield for all the support sofar.
Slowly getting to learn this, but I like it.

**Comment by johnniezen** [comment]
r tomtaylor lc-thomasberger

File: routes/lang/nl-application.json [hunk 4]
```diff
@@ -0,0 +1,356 @@
...
+  "app.forgotpasswordsuccesstitle": "Verzoek gelukt",
+  "app.emailadminpasswordchangesubject": "Uw Adapt wachtwoord is veranderd",
+  "app.emailadminpasswordchangebody": "Het wachtwoord voor uw <a href=\'%{rootUrl}\' target=\'_blank\'>Adapt Authoring tool</a> account is veranderd. Neem aub contact op met de beheerder, als u dit niet verzocht heeft.",
+  "app.role": "Rol",
+  "app.warningdefaulttitle": "Waarschuwing",
+  "app.errorpagenoaccesstitle": "Toegang geweigerd",
+  "app.errorpagenoaccess": "Sorry, u heeft niet de juiste bevoegdheden om deze pagina te zien. Klik op onderstaande knop om naar het dashboard te gaan.",
+  "app.greeting": "Hallo",
+  "app.emaillogotext": "ADAPT",
+  "app.emailfooter": "Verzonden door Adapt Learning.",
```
Comment (lc-thomasberger) [inline routes/lang/nl-application.json]: can you please remove the ending `,`

File: routes/lang/nl-application.json [hunk 5]
```diff
@@ -0,0 +1,355 @@
...
+  "app.adduserformtitle": "Nieuwe gebruikers account details",
+  "app.gotodashboard": "Ga naar het Dashboard",
+  "app.editsettings": "Verander instellingen",
+  "app.editcourse": "Verander cursus",
+  "app.addsubmenu": "Voeg submenu toe",
+  "app.confirmpassword": "Bevestig wachtwoord",
+  "app.passwordtip": "Voer onder een nieuw wachtwoord in. Een veilig wachtwoord heeft een mix van nummers, letters (inclusief hoofdleters) en andere karakters.",
+  "app.forgotpasswordsuccesstitle": "Verzoek gelukt",
+  "app.emailadminpasswordchangesubject": "Uw Adapt wachtwoord is veranderd",
+  "app.emailadminpasswordchangebody": "Het wachtwoord voor uw <a href=\'%{rootUrl}\' target=\'_blank\'>Adapt Authoring tool</a> account is veranderd. Neem aub contact op met de beheerder, als u dit niet verzocht heeft.",
```
Comment (lc-thomasberger) [inline routes/lang/nl-application.json]: Can you also please remove the `\` from this line.

**Comment by taylortom** [comment]
Apologies, closing due to inactivity.

If there's still an appetite for these changes, please log an issue in the [v1](https://github.com/adapt-security/adapt-authoring) codebase.

**Pull request closed without merging**